pkg/iac-providers/kustomize/commons/load-file_test.go

```diff
@@ -13,22 +13,22 @@ func TestLoadIacFile(t *testing.T) {
 		name      string
 		filePath  string
 		options   map[string]interface{}
-		kustomize KustomizeV4
+		kustomize KustomizeDirectoryLoader
 		typeOnly  bool
 		want      output.AllResourceConfigs
 		wantErr   error
 	}{
 		{
 			name:      "load iac file is not supported for kustomize",
 			filePath:  "/dummyfilepath.yaml",
-			kustomize: KustomizeV4{},
+			kustomize: KustomizeDirectoryLoader{},
 			wantErr:   errLoadIacFileNotSupported,
 		},
 	}
 
 	for _, tt := range table {
 		t.Run(tt.name, func(t *testing.T) {
-			_, gotErr := tt.kustomize.LoadIacFile(tt.filePath, tt.options)
+			_, gotErr := tt.kustomize.LoadIacFile()
```

Comment:
since there is only one test and we are expecting a sentinel error back, should we just use `gotErr == errLoadIacFileNotSupported` instead of using `reflect.DeepEqual` ?